CODEOWNERS

```diff
@@ -6,250 +6,250 @@
-soc/x86/intel_quark/quark_d2000/         @nashif
-soc/x86/intel_quark/quark_se/            @nashif
-soc/x86/intel_quark/quark_x1000/         @nashif
-arch/xtensa/                             @andrewboie @rgundi @andyross
-soc/xtensa/                              @andrewboie @rgundi @andyross
-boards/arc/                              @vonhust @ruuddw
-boards/arc/arduino_101_sss/              @nashif
-boards/arc/em_starterkit/                @vonhust
-boards/arc/quark_se_c1000_ss_devboard/   @nashif
-boards/arm/                              @MaureenHelm @galak
-boards/arm/96b_argonkey/                 @avisconti
-boards/arm/96b_carbon/                   @rsalveti @idlethread
-boards/arm/96b_nitrogen/                 @idlethread
-boards/arm/96b_neonkey/                  @Mani-Sadhasivam
-boards/arm/96b_stm32_sensor_mez/         @Mani-Sadhasivam
-boards/arm/cc3220sf_launchxl/            @GAnthony
-boards/arm/curie_ble/                    @jhedberg
-boards/arm/disco_l475_iot1/              @erwango
-boards/arm/frdm*/                        @MaureenHelm
-boards/arm/frdm*/doc/                    @MaureenHelm @MeganHansen
-boards/arm/hexiwear*/                    @MaureenHelm
-boards/arm/hexiwear*/doc/                @MaureenHelm @MeganHansen
-boards/arm/lpcxpresso*/                  @MaureenHelm
-boards/arm/lpcxpresso*/doc/              @MaureenHelm @MeganHansen
-boards/arm/mimxrt*/                      @MaureenHelm
-boards/arm/mimxrt*/doc/                  @MaureenHelm @MeganHansen
-boards/arm/mps2_an385/                   @fvincenzo
-boards/arm/msp_exp432p401r_launchxl/     @Mani-Sadhasivam
-boards/arm/nrf51_blenano/                @rsalveti
-boards/arm/nrf51_pca10028/               @carlescufi
-boards/arm/nrf52_pca10040/               @carlescufi
-boards/arm/nrf52_pca20020/               @tkln
-boards/arm/nrf52810_pca10040/            @carlescufi
-boards/arm/nrf52840_pca10056/            @carlescufi
-boards/arm/nrf52840_pca10059/            @lemrey
-boards/arm/nrf9160_pca10090/             @ioannisg
-boards/arm/nucleo_f401re/                @rsalveti @idlethread
-boards/arm/sam4s_xplained/               @fallrisk
-boards/arm/v2m_beetle/                   @fvincenzo
-boards/arm/olimexino_stm32/              @ydamigos
-boards/arm/stm32f3_disco/                @ydamigos
-boards/nios2/                            @ramakrishnapallala
-boards/nios2/altera_max10/               @ramakrishnapallala
-boards/posix/                            @aescolar
-boards/riscv32/                          @kgugala @pgielda @nategraff-sifive
-boards/x86/                              @andrewboie @nashif
-boards/x86/arduino_101/                  @nashif
-boards/x86/galileo/                      @nashif
-boards/x86/quark_d2000_crb/              @nashif
-boards/x86/quark_se_c1000_devboard/      @nashif
-boards/xtensa/                           @andrewboie @ramakrishnapallala
+/.known-issues/                           @inakypg @nashif
```

Comment:
Could have named the commit "*prepend* / to all entries" or "use absolute paths for all entries".